zio-json/jvm/src/main/scala/zio/JsonPackagePlatformSpecific.scala

```diff
@@ -9,22 +9,22 @@ import java.nio.charset.StandardCharsets
 import java.nio.file.{ Path, Paths }
 
 trait JsonPackagePlatformSpecific {
-  def readJsonAs(file: File): ZStream[Any, Throwable, ast.Json] =
+  def readJsonAs(file: File): ZStream[Scope, Throwable, ast.Json] =
```

Comment:
Ordinarily, streams bundle scopes within them, so there is no need to leak `Scope` on the outside. 

The only exception is if you want the resources consumed by the stream to stay open beyond the lifetime of the stream.

What do you think here and elsewhere?